dev/attest/package.json

```diff
@@ -30,6 +31,9 @@
     },
     "scripts": {
         "build": "ts-node ../scripts/build.ts",
+        "clean": "pnpm run \"/^clean:.*/\"",
```

Comment:
I think the build script does this by default. It also looks like there's a type in `node_modules`, and IIRC Windows still doesn't like rm -rf in CI, so maybe we can just remove these for now?